subliminal/core.py

```diff
@@ -357,6 +357,49 @@ def search_external_subtitles(path, directory=None):
     return subtitles
 
 
+def is_valid_file(path):
```

Comment:
This is misleading. The name of the method leads anyone to think that it returns true or false and the method implemented doesn't behave like that